### PR TITLE
Fix file path on cookbook 'add property to category'

### PR DESCRIPTION
**Pull request by pierallard**
Fix file not found at the bottom fo this page and improve some doc.
http://docs.akeneo.com/latest/cookbook/category/add-properties-to-category.html#define-the-category-form

### CODE REVIEW

**Comment by solivier** [comment]
gtm

**Comment by juliensnz** [comment]
GTM

File: src/Acme/Bundle/CatalogBundle/Resources/config/validation/category.yml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-Acme\CatalogBundle\Entity\MyCategory:
+Acme\CatalogBundle\Entity\Category:
```
Comment (skeleton) [inline src/Acme/Bundle/CatalogBundle/Resources/config/validation/category.yml]: `Acme\Bundle\CatalogBundle\Entity\Category`

**Comment by skeleton** [comment]
1 comment then GTM

**Pull request merged**